addons/controllers/antrea/antreaconfig_controller.go

```diff
@@ -167,10 +194,189 @@ func (r *AntreaConfigReconciler) ReconcileAntreaConfigNormal(
 		return err
 	}
 
+	if antreaConfig.Spec.AntreaNsx.BootstrapFrom.ProviderRef != nil && antreaConfig.Spec.AntreaNsx.BootstrapFrom.Inline != nil {
+		err := fmt.Errorf("providerRef and inline should not be both set in AntreaConfig.spec.antreaNsx.bootstrapFrom")
+		antreaConfig.Status.Message = err.Error()
+	}
+	antreaConfig.Status.Message = ""
```

Comment:
why settings this message to empty is necessary? if both fields are set, for example, we lose the error.